server/src/test/kotlin/com/github/neho4y/comment/service/CommentServiceTest.kt

```diff
@@ -0,0 +1,69 @@
+package com.github.neho4y.comment.service
+
+import com.github.neho4y.comment.assertEquals
+import com.github.neho4y.comment.createDefaultComment
+import com.github.neho4y.comment.createDefaultCommentCreationDto
+import com.github.neho4y.feedback.domain.Feedback
+import com.github.neho4y.feedback.domain.repository.FeedbackRepository
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.annotation.DirtiesContext
+
+@SpringBootTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+internal class CommentServiceIntegrationTest {
+    @Autowired
+    private lateinit var commentService: CommentService
+
+    @Autowired
+    private lateinit var feedbackRepository: FeedbackRepository
+    private var feedbackId: Long = 0
+
+    @BeforeEach
+    fun createCommentService() {
+        val feedback = Feedback()
+        feedbackRepository.save(feedback)
+        feedbackRepository.flush()
+        feedbackId = feedback.id
+    }
+
+    @Test
+    fun `When comment is added then it can be found`() {
+        val feedback = feedbackRepository.getOne(feedbackId)
+        val commentCreationDto = createDefaultCommentCreationDto(feedback.id)
+        commentService.addComment(commentCreationDto)
+        assertDoesNotThrow {
+            val comments = commentService.getComments(commentCreationDto.feedbackId)
+            assertNotNull(comments)
+            assertTrue(comments.size == 1)
+            comments[0].assertEquals(createDefaultComment(feedback))
+        }
+    }
+
+    @Test
+    fun `When deleted then deleted`() {
+        val feedback = feedbackRepository.getOne(feedbackId)
+        val commentCreationDto = createDefaultCommentCreationDto(feedback.id)
+        commentService.addComment(commentCreationDto)
+        val comment = commentService.getComments(commentCreationDto.feedbackId)[0]
```

Comment:
You may use `getComments(...).first()` instead of accessing first element with `[0]` here and in some other places